modules/asoBidAdapter.md

```diff
@@ -14,12 +14,11 @@ For more information, please visit [Adserver.Online](https://adserver.online).
 
 # Parameters
 
-| Name          | Scope    | Description             | Example   | Type      |
-|---------------|----------|-------------------------|-----------|-----------|
-| `zone`        | required | Zone ID                 | `73815`   | `Integer` |
-| `attr`        | optional | Custom targeting params | `{keywords: ["a", "b"]}` | `Object` |
-
-
+| Name      | Scope    | Description             | Example                  | Type       |
+|-----------|----------|-------------------------|--------------------------|------------|
+| `zone`    | required | Zone ID                 | `73815`                  | `Integer`  |
+| `attr`    | optional | Custom targeting params | `{keywords: ["a", "b"]}` | `Object`   |
```

Comment:
Please note that if you take keywords from bid parameters, you must also take them from ortb2 object